### PR TITLE
Deprecate cluster-density

### PR DESCRIPTION
**Pull request by rsevilla87**
## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [x] Optimization
- [x] Documentation Update

## Description

Deprecating the original cluster-density workload. We should use cluster-density-v2 in all our tests, this workload is more relevant and complete.

Changes will be applied in the next kube-burner major version

cc: @paigerube14 @jtaleric @dry923 @afcollins @svetsa-rh @krishvoor 

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

### CODE REVIEW

**Review by jtaleric — APPROVED** [review]
lgtm

**Review by krishvoor — APPROVED** [review]
/lgtm

**Review by vishnuchalla — APPROVED** [review]

**Review by afcollins — APPROVED** [review]
LGTM!

File: docs/ocp.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Usage:
   kube-burner ocp [command]
 
 Available Commands:
-  cluster-density    Runs cluster-density workload
+  cluster-density    Runs cluster-density workload. Deprecated
```
Comment (smalleni) [inline docs/ocp.md:14]: This is total removal, not deprecation so I don't believe the help should include this any more.

**Review by smalleni — CHANGES_REQUESTED** [review]
This is total removal, not deprecation so I don't believe the help should include this any more.

File: docs/ocp.md (hunk 1 above)
Comment (rsevilla87) [reply docs/ocp.md]: fixed

**Pull request merged**